jobs/mysql/templates/drain.sh

```diff
@@ -1,6 +1,59 @@
-#!/bin/bash -e
+#!/bin/bash -eu
 
-/var/vcap/packages/mariadb/support-files/mysql.server stop --pid-file=/var/vcap/sys/run/mysql/mysql.pid > /dev/null
-return_code=$?
-echo 0
-exit ${return_code}
+NODE_IP=<%= spec.ip %>
+MYSQL_PORT=<%= p("cf_mysql.mysql.port") %>
+
+LOG_DIR="/var/vcap/sys/log/mysql/"
```

Comment:
Please remove this line and replace this with the logging pattern demonstrated in https://github.com/gstackio/exemplar-release/blob/latest/jobs/paragon/templates/drain

Then your `echo "..." &>> "$LOG_DIR/drain.log"` statements will become plain `echo "..."` because of the default redirection, and exit protocol (status and echoing integer) will be handled automatically at script exit.